e2e/benchmarks/browserstack-benchmark/browser_list.json

```diff
@@ -0,0 +1,490 @@
+[
+  {
+    "os": "Windows",
+    "os_version": "10",
+    "browser": "chrome",
+    "device": null,
+    "browser_version": "84.0",
+    "real_mobile": null
+  },
+  {
+    "os": "Windows",
+    "os_version": "10",
+    "browser": "ie",
+    "device": null,
+    "browser_version": "11.0",
+    "real_mobile": null
+  },
+  {
+    "os": "Windows",
+    "os_version": "10",
+    "browser": "edge",
+    "device": null,
+    "browser_version": "84.0",
+    "real_mobile": null
+  },
+  {
+    "os": "Windows",
+    "os_version": "10",
+    "browser": "firefox",
+    "device": null,
+    "browser_version": "79.0",
+    "real_mobile": null
+  },
+  {
+    "os": "OS X",
+    "os_version": "Catalina",
+    "browser": "chrome",
+    "device": null,
+    "browser_version": "84.0",
+    "real_mobile": null
+  },
+  {
+    "os": "OS X",
+    "os_version": "Catalina",
+    "browser": "safari",
+    "device": null,
+    "browser_version": "13.1",
+    "real_mobile": null
+  },
+  {
+    "os": "OS X",
+    "os_version": "Catalina",
+    "browser": "firefox",
+    "device": null,
+    "browser_version": "79.0",
+    "real_mobile": null
+  },
+  {
+    "os": "ios",
+    "os_version": "13",
+    "browser": "iphone",
```

Comment:
can the browser type be specified for iphone/ipad or android? maybe as safari or chrome?

Reply:
No, currently it is not tunable for mobile devices. (for android, the browser is Chrome; for ios, the browser is Safari)